pwndbg/commands/version.py

```diff
@@ -67,5 +67,116 @@ def version():
         ida_py_ver  = 'IDA Py:   %s' % ida_versions['python']
         ida_hr_ver  = 'Hexrays:  %s' % ida_versions['hexrays']
         all_versions += (ida_version, ida_py_ver, ida_hr_ver)
+    return all_versions
+    
+
+@pwndbg.commands.Command
+def version():
+    """
+    Displays gdb, python and pwndbg versions.
+    """
+    print('\n'.join(map(message.system, all_versions())))
+
+
+bugreport_parser = argparse.ArgumentParser(description='''
+    Generate bugreport
+    ''')
+bugreport_parser.add_argument('--browse', '-b', action='store_true', help='Open browser on github/issues/new')
+
+@pwndbg.commands.ArgparsedCommand(bugreport_parser)
+def bugreport(browse=False):
+    ISSUE_TEMPLATE = '''
+<!--
+Before reporting a new issue, make sure that we do not have any duplicates already open.
+If there is one it might be good to take part in the discussion there.
+
+Please make sure you have checked that the issue persists on LATEST pwndbg version.
+
+Below is a template for BUG REPORTS.
+Don't include it if this is a FEATURE REQUEST.
+-->
 
-    print('\n'.join(map(message.system, all_versions)))
+
+### Description
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+### Steps to reproduce
+
+<!--
+What do we have to do to reproduce the problem?
+If this is connected to particular C/asm code, 
+please provide the smallest C code that reproduces the issue.
+-->
+
+Gdb session history:
+{gdb_history}
+
+### My setup
+
+<!--
+Show us your gdb/python/pwndbg/OS/IDA Pro version (depending on your case).
+
+NOTE: We are currently supporting only Ubuntu installations.
+It is known that pwndbg is not fully working e.g. on Arch Linux (the heap stuff is not working there).
+If you would like to change this situation - help us improving pwndbg and supporting other distros!
+
+This can be displayed in pwndbg through `version` command.
+
+If it is somehow unavailable, use:
+* `show version` - for gdb
+* `py import sys; print(sys.version)` - for python
+* pwndbg version/git commit id
+-->
+
+{setup}'''
+
+    gdb_config = gdb.execute('show configuration', to_string=True).split('\n')[:-3]
+    all_info = all_versions()
+
+    current_setup = 'Platform: %s\n' % platform()
+    current_setup += '\n'.join(all_info)
+    current_setup += '\n'.join(gdb_config)
+
+    # get saved history size (not including current gdb session)
+    gdb_history_file = gdb.execute('show history filename', to_string=True)
+    gdb_history_file = gdb_history_file[len('The filename in which to record the command history is "'):-3]
+    gdb_history_len = 0
+    try:
+        with open(gdb_history_file, 'r') as f:
+            gdb_history_len = len(f.readlines())
+    except FileNotFoundError:
+        pass
+
+    max_command_no = int(gdb.execute('show commands', to_string=True).split('\n')[-2].split('  ')[1]) - 1
```

Comment:
We shouldn't rely on whitespace width. I guess this might fail if we had like 1000+ entries in commands history:
```python
gdb.execute('show commands', to_string=True).split('\n')[-2].split('  ')
```

This can be solved by not providing any argument to `.split`, so it will split on whitespaces no matter of their length (and remove them from output), as stated in the docs:
```
In [1]: str.split?
Docstring:
S.split([sep [,maxsplit]]) -> list of strings

Return a list of the words in the string S, using sep as the
delimiter string.  If maxsplit is given, at most maxsplit
splits are done. If sep is not specified or is None, any
whitespace string is a separator and empty strings are removed
from the result.
```